src/core/sys/linux/perf_event.d

```diff
@@ -2073,8 +2073,6 @@ else
 
                 struct
                 {
-                        import std.bitmanip : bitfields;
-
                         /* mixin(bitfields!(ulong, "mem_rsvd", 24, ulong, "mem_snoopx", 2, ulong,
```

Comment:
I would have just moved the `/*` before the `import`, as the `bitfields` comment is just for easier maintenance (if you need to regenerate the code for some reason).